### PR TITLE
[DEV-1677]: add quote component

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

Add `Quote` component

#### Motivation and Context

The new solutions page added a new component called `Quote`

#### How Has This Been Tested?

Storybook story

#### Screenshots (if appropriate):
Screenshot 2024-05-29 alle 08 17 55

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/atoms/Quote/Quote.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            sx={{
+              color: textColor,
+              alignSelf: 'flex-end',
+              position: 'relative',
+              left: '24px',
+            }}
+          />
+        </Stack>
+      </Stack>
+      {backgroundImage && (
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/Quote/Quote.tsx:64]: The `backgroundImage` has a default value, is this condition necessary?

File: apps/nextjs-website/src/components/atoms/Quote/Quote.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      >
+        <Stack
+          px={{ xs: 4, md: 18 }}
+          direction={'column'}
+          justifyContent={'start'}
+          zIndex={10}
+          sx={{ width: 'fit-content', maxWidth: '40%' }}
+        >
+          <FormatQuote sx={{ color: textColor, rotate: '180deg' }} />
+          <Typography color={textColor} variant='body1'>
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/Quote/Quote.tsx:51]: The `variant` is not corresponding to the design in Figma
image

File: apps/nextjs-website/src/components/atoms/Quote/Quote.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        justifyContent={'start'}
+        alignItems={'center'}
+        direction={'row'}
+      >
+        <Stack
+          px={{ xs: 4, md: 18 }}
+          direction={'column'}
+          justifyContent={'start'}
+          zIndex={10}
+          sx={{ width: 'fit-content', maxWidth: '40%' }}
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/Quote/Quote.tsx:48]: I think that on small devices the quote should occupy more than 40% of the screen width. Ask the design team for advice

image

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/atoms/Quote/Quote.tsx (hunk 3 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/atoms/Quote/Quote.tsx:48]: I asked the design team for advice

**Review by marcobottaro — CHANGES_REQUESTED** [review]
image
Compared to Figma, the quote symbol is smaller, it's white (it should be `#e0e0e0`) and it's too close to the text

**Pull request merged**